app/views/samples/show_amr_counts.html.erb

```diff
@@ -0,0 +1,7 @@
+<div id="AMRView_page">
```

Comment:
Since we have the tab now, we don't need this page anymore. Please remove it

Reply:
done